regulations/static/regulations/js/source/views/main/child-view.js

```diff
@@ -82,7 +82,7 @@ var ChildView = Backbone.View.extend({
     // for the fixed position header, is deemed the active section
     checkActiveSection: function() {
       $.each(this.$sections, function(idx, $section) {
-        if ($section.offset().top + this.$contentHeader.height() >= $(window).scrollTop()) {
+        if ($section.offset().top >= $(window).scrollTop()) {
```

Comment:
After an out-of-band chat, it looks like we want to add an offset of 30-50px (roughly one line of text) to keep previous functionality. We'll add a ticket around changing the logic to give a better user experience.